docs/FAQ.md

```diff
@@ -11,12 +11,16 @@ brand.
 
 ### Is Backstage a monitoring platform?
 
-No, but it can be! Backstage is designed to be a developer portal for all your infrastructure tooling, services, and documentation. So, it's not a monitoring platform — but that doesn't mean you can't integrate a monitoring tool into Backstage by writing [a plugin](https://github.com/spotify/backstage/blob/master/docs/FAQ.md#what-is-a-plugin-in-backstage).
+No, but it can be! Backstage is designed to be a developer portal for all your
```

Comment:
Apparently, prettier didn't like this particular file and decided to process it during a precommit hook :)